### DISCUSSION TITLE
DotSpatial and NetCDF

### DISCUSSION THREAD

**Discussion by EwEDevTeam** [DotSpatial/DotSpatial#1468] [Q&A]
Dear all,

To what degree does DotSpatial already support NetCDF? Are there any working examples that I can take a look at? Google doesn't know. 
There is a strong need in the marine ecology domain to read & write NetCDF data through DotSpatial, and I put time into coding to streamline the workflow.

Thx,
Jeroen

**Comment by jany-tenaj** [comment]
Check out whether NetCDF is supported by GDAL. If so you should be able to write something to open NetCDF via GDAL in DotSpatial.
